nbbuild/antsrc/org/netbeans/nbbuild/extlibs/ignored-overlaps

```diff
@@ -59,11 +59,18 @@ enterprise/web.core.syntax/external/struts-tiles-1.3.10.jar enterprise/web.strut
 groovy/gradle/external/gradle-4.10.2-bin.zip ide/c.google.gson/external/gson-2.7.jar
 groovy/gradle/external/gradle-4.10.2-bin.zip ide/c.jcraft.jsch/external/jsch-0.1.54.jar
 groovy/gradle/external/gradle-4.10.2-bin.zip ide/o.apache.commons.lang/external/commons-lang-2.6.jar
-groovy/gradle/external/gradle-4.10.2-bin.zip java/maven/external/aether-api-1.13.1.jar
-groovy/gradle/external/gradle-4.10.2-bin.zip java/maven.embedder/external/apache-maven-3.3.9-bin.zip
+groovy/gradle/external/gradle-4.10.2-bin.zip java/maven.embedder/external/apache-maven-3.6.3-bin.zip
 groovy/gradle/external/gradle-4.10.2-bin.zip platform/api.annotations.common/external/jsr305-3.0.2.jar
 groovy/gradle/external/gradle-4.10.2-bin.zip platform/libs.junit4/external/hamcrest-core-1.3.jar
 groovy/gradle/external/gradle-4.10.2-bin.zip platform/libs.junit4/external/junit-4.12.jar
+groovy/gradle/external/gradle-4.10.2-bin.zip java/maven/external/aether-api-1.13.1.jar
+
+#
+#gradle-4.10.2/lib/javax.inject-1.jar in groovy/gradle/external/gradle-4.10.2-bin.zip and apache-maven-3.6.3/lib/javax.inject-1.jar in 
+#java/maven.embedder/external/apache-maven-3.6.3-bin.zip are identical
+#gradle-4.10.2/lib/plugins/plexus-cipher-1.7.jar in groovy/gradle/external/gradle-4.10.2-bin.zip and apache-maven-3.6.3/lib/plexus-cipher-1.7.jar in java/maven.embedder/external/apache-maven-3.6.3-bin.zip are identical</failure>
```

Comment:
This looks strange - if the ignore is needed, it should be present, if it is a comment, please remove it.

Reply:
no this is bad copy paste sorry